critics.xml

```diff
@@ -29,6 +29,10 @@
       <description></description>
     </class>
 
+    <class type="mppi::critics::TwirlingCritic" base_class_type="mppi::critics::CriticFunction">
+      <description></description>
```

Comment:
Add description: mppi critic for preventing twirling behavior when using omnidirectional models

Reply:
Done in new PR